iOS/App/Lorem.swift

```diff
@@ -81,56 +81,6 @@ class Lorem {
                             handle: String())
     }
 
-    class func systemConversation() -> SystemConversation {
```

Comment:
We no longer need system conversations because Stream makes it easy to mock data. I've deleted all references to it.

Reply:
sick